src/main/java/com/sourcegraph/cody/agent/CodyAgentClient.java

```diff
@@ -62,4 +64,34 @@ public void chatUpdateMessageInProgress(ChatMessage params) {
   public void debugMessage(DebugMessage msg) {
     logger.warn(String.format("%s: %s", msg.getChannel(), msg.getMessage()));
   }
+
+  // Webviews
+  @JsonRequest("webview/create")
+  public CompletableFuture<Void> webviewCreate(WebviewCreateParams params) {
+    logger.error("webview/create This request should not happen if you are using chat/new.");
+    return CompletableFuture.completedFuture(null);
+  }
+
+  @JsonNotification("webview/postMessage")
+  public void webviewPostMessage(WebviewPostMessageParams params) {
+    if (onChatUpdateMessageInProgress != null
+        && params.getMessage().getType().equals("transcript")) {
```

Comment:
Can we replace hardcoded string `"transcript"` with some const?

Reply:
done